subprojects/spdk.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/spdk/spdk.git
-revision = v21.10
+revision = v22.05
```

Comment:
I think the change from SPDK 21 to SPDK 22 should be a separate PR, I think it is worth it to make #148 as simple as possible

Reply:
I realize now that it was originally a separate PR (#149), and that was based on this.
Maybe it should be the other way around?

Reply:
The upgrade to v22.05 requires the "tooling makeover".
Moved the commit to front.

I guess I should have left the commit at PR #149 but now that it is here and were just nit-picking, how about we just leave it be?

Reply:
Sure, that is fine by me